v1/plugin/plugin.go

```diff
@@ -280,67 +342,70 @@ func buildGRPCServer(typeOfPlugin pluginType, name string, version int, opts ...
 // generates a response for the initial stdin / stdout handshake, and starts
 // the plugin's gRPC server.
 func StartCollector(plugin Collector, name string, version int, opts ...MetaOpt) int {
-	server, m, err := buildGRPCServer(collectorType, name, version, opts...)
+	appArgs.plugin = plugin
+	appArgs.name = name
+	appArgs.version = version
```

Comment:
Added in c6be777

![img](https://www.evernote.com/l/AA06QQsV36JJgoqY2X5cHii98UEubjhvxlcB/image.png)